.circleci/config.yml

```diff
@@ -337,28 +337,80 @@ jobs:
            echo "${DOCKER_PASS:?}" | \
             docker login -u "${DOCKER_USER:?}" --password-stdin
             docker push "$IMAGE"
-  deploy-to-private-gcr:
-    executor: gcp-gcr/default
+  private-generate-sql:
+    docker: *docker
     steps:
       - checkout
-      - attach_workspace:
-          at: /tmp/generated-sql
+      - *restore_venv_cache
+      - *build
+      - *restore_mvn_cache
+      - *java_deps
       - add_ssh_keys:
           # deploy key to private-bigquery-etl
-          fingerprints: "5b:7b:00:df:da:e1:9f:07:c7:69:60:1a:3f:16:a5:ed"
+          fingerprints: "cf:d6:25:9a:ee:26:66:39:c8:cc:48:f6:bb:3e:34:68"
       - run:
-          name: Move generated-sql into place
+          name: Install rsync
           command: |
-            rm -rf sql/
-            cp -r /tmp/generated-sql/sql sql
+            apt update
+            apt install -y rsync
       - run:
-          name: Move private sql into place
+          name: Pull down private SQL content
           command: |
             ssh-keyscan -t rsa github.com >> ~/.ssh/known_hosts
             git clone --single-branch --branch main \
                           git@github.com:mozilla/private-bigquery-etl.git \
                           ~/private-bigquery-etl
             rsync --archive --ignore-existing ~/private-bigquery-etl/sql/ sql/
+      - run:
+          name: Generate SQL content
+          command: |
+            mkdir /tmp/private-generated-sql
```

Comment:
It would be nice to be able to factor out the logic for the generate-sql and push-generated-sql jobs so that there's less duplication with the private jobs, but my gut is telling me the duplication here is likely more maintainable than the code would be refactored, since we'd have to figure out how to templatize the paths involved.

Reply:
Yes, I agree, we're getting some duplication here and in some parts of private repo's config. In the future we can think of using inline orbs to fix this.